docs/how-to-guides/effective-gno.md

```diff
@@ -0,0 +1,255 @@
+---
+id: 'effective-gno'
+---
+
+# Effective Gno
+
+Welcome to the guide for writing effective Gno code. This document is designed
+to help you understand the nuances of Gno and how to use it effectively.
+
+Before we dive in, it's important to note that Gno shares several similarities
+with Go. Therefore, if you haven't already, we highly recommend reading
+["Effective Go"](https://go.dev/doc/effective_go) as a primer.
+
+## Counter-Intuitive Good Practices
+
+This section highlights some Gno good practices that might seem
+counter-intuitive, especially if you're coming from a Go background.
+
+### Embrace Global Variables in Gno
+
+In Gno, using global variables isn't just acceptable - it's encouraged. This is
```

Comment:
Perhaps qualify that using global variables is preferred _only_ because of realm persistence -- just to guard against someone misunderstanding and making all their variables global.

Reply:
ec37c80e